ui/src/store/resource.ts

```diff
@@ -41,7 +41,8 @@ const Version = types.model('Version', {
 export const Resource = types
   .model('Resource', {
     id: types.number,
-    name: types.identifier,
+    name: types.string,
+    uniqueKey: types.identifier,
```

Comment:
instead of `uniqueKey` can we rename it to a better variable like `resourceKey`?